h/services/user_delete.py

```diff
@@ -38,29 +38,29 @@ def delete_user(self, user: User, requested_by: User, tag: str):
         # request times out.
         #
         # So add a job to purge the user's data in the background.
-        self.db.add(
-            Job(
-                name=Job.JobName.PURGE_USER,
-                priority=0,
-                tag="UserDeleteService.delete_user",
-                kwargs={"userid": user.userid},
-            )
+        job = Job(
+            name=Job.JobName.PURGE_USER,
+            priority=0,
+            tag="UserDeleteService.delete_user",
+            kwargs={"userid": user.userid},
         )
+        self.db.add(job)
+        log.info("Added purge_user job: %r", job)
```

Comment:
FIXME: Need to flush the DB before logging this to generate default values for the job (e.g. its id).